templates/header.war.html

```diff
@@ -11,12 +11,24 @@
 <p>
   You will need to pre-install a supported version of Java:
 <p>
-
-<ul>
-  <li>2.164 (2019-02) and newer: Java 8 or Java 11</li>
-  <li>2.54 (2017-04) and newer: Java 8</li>
-  <li>1.612 (2015-05) and newer: Java 7</li>
+	      <b>Weekly Version</b>
```

Comment:
"Weekly Version" does not really fit existing terminology use. On https://www.jenkins.io/download/ and elsewhere, we call these _release lines_, the same terminology should be used here.